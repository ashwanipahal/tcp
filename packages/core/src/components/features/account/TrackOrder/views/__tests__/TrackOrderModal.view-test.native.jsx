import React from 'react';
import { shallow } from 'enzyme';
// eslint-disable-next-line import/no-named-as-default
import TrackOrderModal from '../TrackOrderModal.view.native';

describe('TrackOrderModal component', () => {
  it('should renders correctly', () => {
    const props = {
      labels: {
        trackOrder: {},
      },
      openState: true,
      setModalMountState: jest.fn(),
      onSubmit: jest.fn(),
      errorMessage: null,
      showNotification: 'error',
      onChangeForm: jest.fn(),
      handleToggle: jest.fn(),
    };
    const component = shallow(<TrackOrderModal {...props} />);
    expect(component).toMatchSnapshot();
  });
});
