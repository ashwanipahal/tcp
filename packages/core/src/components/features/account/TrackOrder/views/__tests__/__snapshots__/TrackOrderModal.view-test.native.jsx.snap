// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackOrderModal component should renders correctly 1`] = `
<ModalNative
  fontSize="fs22"
  headerStyle={
    Object {
      "width": "100%",
    }
  }
  headingAlign="center"
  headingFontFamily="secondary"
  horizontalBar={false}
  isOpen={true}
  onRequestClose={[Function]}
>
  <SafeAreaView>
    <TrackOrderView
      errorMessage={null}
      handleToggle={[MockFunction]}
      labels={
        Object {
          "trackOrder": Object {},
        }
      }
      onChangeForm={[MockFunction]}
      onSubmit={[MockFunction]}
      setModalMountState={[MockFunction]}
      showNotification="error"
    />
  </SafeAreaView>
</ModalNative>
`;
